ops/docker/Dockerfile.packages

```diff
@@ -17,68 +17,75 @@ FROM alpine:3.16 as manifests
 RUN apk add coreutils
 
 WORKDIR /tmp
-COPY yarn.lock .nvmrc package.json ./src/
+COPY pnpm-lock.yaml pnpm-workspace.yaml .nvmrc package.json ./src/
 COPY packages src/packages/
 RUN mkdir manifests && \
   cd src && \
   # copy package.json recursively
   find . -name 'package.json' | xargs cp --parents -t ../manifests/ && \
-  # yarn.lock
-  cp yarn.lock ../manifests/ && \
+  # pnpm-lock.yaml
+  cp pnpm-lock.yaml ../manifests/ && \
+  # pnpm-workspace.yaml
+  cp pnpm-workspace.yaml ../manifests/ && \
   # .nvmrc
   cp .nvmrc ../manifests/
 
 FROM ethereumoptimism/foundry:latest as foundry
-FROM node:16-alpine3.14 as base
+# bullseye-slim is debian based
+# we use it rather than alpien because it's not much
+# bigger and alpine is often missing packages for node applications
+# alpine is not officially supported by node.js
+FROM node:16.16.0-bullseye-slim as base
```

Comment:
This should be in it's own PR and reviewed by zach and any security folks.   This is a pretty big change to use debian instead of alpine.

From my experience and also gathering the wisdom of the crowd building js docker images, because node.js doesn't support alpine officially and doesn't recomend using it, and because I and many others have struggled in past mantaining alpine builds, it's not worth using alpine. 

Negative tradeoff

- bullseye-slim is less minimal than alpine so slightly bigger surface area for security vulnerabilities.   It's an extremely common and battle tested image though
- bullseye-slim is bigger.  It's not significantly bigger though and that is offset a bit by doing a multistage build

Reply:
Could likely get it to work on alpine but it would take a long time to figure out or a hack (simple hack is not running postinstall scripts at all which would work but is a hack and could cause hard to debug issues in future)

Reply:
All node.js images should use a debian image by defaut.   If we needed an image as tiny and minimal as alpine, there are better alternatives such as copying into [gcr.io/distroless/nodejs18](https://github.com/roninjin10/stax/blob/main/Dockerfile#L242)